extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -74,22 +70,17 @@ private function getInsightModule( $type, $config ) {
 		 *
 		 * communitypage-popularpages-title'
 		 * communitypage-uncategorizedpages-title'
-		 * communitypage-wantedpages-title'
 		 * communitypage-deadendpages-title'
-		 * communitypage-wantedpages-description'
 		 */
 		$insightPages['type'] = $type;
 		$insightPages['title'] = wfMessage( 'communitypage-' . $type . '-title' )->text();
-		$insightPages['description'] =  $type === 'wantedpages' ?
-			wfMessage( 'communitypage-wantedpages-description' )->text() :
-			null;
 
-		if ( $insightPages['count'] > self::INSIGHTS_MODULE_ITEMS ) {
+		if ( $insightPages['count'] > static::INSIGHTS_MODULE_ITEMS ) {
 			$insightPages['fulllistlink'] = SpecialPage::getTitleFor( 'Insights', $type )
-				->getLocalURL( $this->getSortingParam( $config[self::INSIGHTS_CONFIG_SORT_TYPE_KEY] ) );
+				->getLocalURL( $this->getSortingParam( $config[static::INSIGHTS_CONFIG_SORT_TYPE_KEY] ) );
 		}
 
-		$insightPages = $this->addLastRevision( $insightPages, $config[self::INSIGHTS_CONFIG_PAGEVIEWS_KEY] );
+		$insightPages = $this->addLastRevision( $insightPages, $config[static::INSIGHTS_CONFIG_PAGEVIEWS_KEY] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$insightPages". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)